readme.md

```diff
@@ -2795,6 +2795,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 | ------: | -------: | -----: |
 |   2 - 6 |      12+ |90-120m |
 
+### [The Gilded Realms](https://boardgamegeek.com/boardgame/397099/gilded-realms)
+
+> The Gilded Realms is a euro-style kingdom builder board game set in a medieval fantasy world, where players will be tasked with ruling over their young but ambitious kingdoms. In this hybrid simultaneous turn-based game of economic management and risk assessment, you and up to three other players will command a unique Faction and compete for the title of ultimate regent.Through careful planning of two builds a turn and management of your resources you must develop your five provinces and construct a thriving income, all the while, fending off competing kingdoms and the invading Sarrukar forces with your armies and fortifications. Players can also earn Gold and Gems from successful caravan trading and management of their kingdom’s stance during economic events. Or they can send armies into other player’s Hinterlands to raid their Caravans and take Endeavour cards for themselves! At the end of the 12th round of play, players earn prestigious titles based on their Victory Points, the highest earning the Serpens Champion and declared winner of The Gilded Realms!
+
+![The Gilded Realms](https://cf.geekdo-images.com/gWNSCaB9gE-cPNHhH33uQQ__thumb/img/zD1zzFlilv3QO_BNwyTDHJpKpJc=/fit-in/200x150/filters:strip_icc()/pic7666055.jpg)
```

Comment:
This images is quite small, please find one in the ~250px size range for both being not large but still being useful with visible text.

Reply:
Hi @edm00se Thank you so much for giving the Feedback. I have changed the image size as mentioned by you. Please check and merge.
Also, I have to complete 2 more PRs by tomorrow end for Hacktoberfest. I have already identified 2 more games to be added. Once this PR is merged, will raise another PR to add another game. Please see if you can merge this fast, so that I am able to complete 2 more PRs by deadline tomorrow.
Thanks